scripts/annotate.py

```diff
@@ -0,0 +1,42 @@
+
+import pandas as pd
+from pycytominer.annotate import annotate
+
+
+
+def annotate_cells(aggregated_data, platemaps):
+    """Annoates
+
+    Parameters
+    ----------
+    aggregated_data : str
+        path pointing to aggregated dataset
+    platemaps : str
+        path pointing to platemaps
+
+    """
+
+    # input paths retrived by snakemake
+    annotated_outfile = str(snakemake.output)
+    platemap_df = pd.read_csv(barcodes)
+
+    # annotating the aggregated profiles
+    # NOTE: join_on should be in config file
```

Comment:
let's delete these superfluous notes - will be good to maintain good documentation from beginning to end of project

Reply:
Unnecessary notes removed
985425f0289ba087a7582e3bbf4eb5ed5786c73a